src/Adapter/Supplier/SupplierOrderValidator.php

```diff
@@ -26,22 +26,31 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Supplier;
```

Comment:
Could you use the recommended method for deprecations : https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations ?